gcexport.py

```diff
@@ -1232,7 +1236,10 @@ def main(argv):
     event_type_name = load_properties(event_type_props)
 
     activities = fetch_activity_list(args, total_to_download)
-    action_list = annotate_activity_list(activities, args.start_activity_no, exclude_list)
+
+    type_filter = list(map(int,args.type_filter.split(',')))  if args.type_filter is not None else None
```

Comment:
Do we need a list here? Based on the code, this looks like an unnecessary memory consumption.

```suggestion
    type_filter = map(int,args.type_filter.split(','))  if args.type_filter is not None else None
```

Reply:
Yep agreed. I guess maybe the more Pythonic approach would be a list comprehension instead but I think it's still pretty clear.